sonar-kotlin-plugin/build.gradle.kts

```diff
@@ -99,7 +99,7 @@ tasks.shadowJar {
     exclude("org/jetbrains/kotlin/org/jline/**")
     exclude("org/jetbrains/kotlin/net/jpountz/**")
     doLast {
-        enforceJarSizeAndCheckContent(shadowJar.get().archiveFile.get().asFile, 33_400_000L, 37_000_000L)
+        enforceJarSizeAndCheckContent(shadowJar.get().archiveFile.get().asFile, 33_400_000L, 37_500_000L)
```

Comment:
Could you, please, check, what caused the difference?
Also, It's important to keep boundaries close to the real value. When moving the upper one, we need to update the lover one too.

Reply:
With the new limit on `master` I don't need this change anymore.